students/g_rama/lesson09/Donor.py

```diff
@@ -0,0 +1,46 @@
+from Donors import Donors
+from CLI import CLI
+
+
+class Donor(Donors):
```

Comment:
what is the relationship here?

Reply:
`Donor` object should represent a donor - methods defined below don't represent that.

Reply:
All these methods are for individual Donor and hence present on Donor class. In my code all the methods related to one donor are present on this class. Any thing related to multiple donors present in donor collection. All input is in CLI class. And the actual calls are in main class

Reply:
Per assignment instructions there should be no user interaction code in "data classes".

Below thank_you method calls CLI method that calls input() function - that's user interaction code.

Reply:
Moved the CLI interaction for thank_you function to mailroom_oob class.